.github/workflows/build-test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           java-version: '11'
           distribution: 'temurin'
       - name: Build with Maven
-        run: mvn clean install -DskipTests -Dci --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+        run: mvn clean install -DskipTests -Dci --batch-mode -D"org.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener"="warn"
```

Comment:
was this not working?

Reply:
Do we need the smoke test also to have a Windows run? In windows the `"` is required for it to run. Currently the smoke test only runs on Linux so we can remove this change and be fine.